pydata/static/import_person.js

```diff
@@ -0,0 +1,38 @@
+function import_from_url(url) {
```

Comment:
@pbanaszkiewicz 
I wish we could have some sort of inheritance in javascript as well.